src/rules/media-feature-name-no-unknown/README.md

```diff
@@ -0,0 +1,73 @@
+# media-feature-name-no-unknown
+
+Disallow unknown media feature names.
+
+```css
+@media (min-width: 700px) { }
+/**     ↑
+ * These media feature names */
+```
+
```

Comment:
Please add a comment about vendor prefixes, like in `property-no-unknown` and `selector-pseudo-class-no-unknown`.